buildpipeline/pipeline.json

```diff
@@ -155,6 +155,24 @@
             "SubType": "netcoreapp"
           }
         },
+		{
+          "Name": "DotNet-CoreFx-Trusted-Windows",
```

Comment:
I see that in your commits you did a cleanup to instead of adding this new blocks using TargetQueues, perhaps you forgot this one.